source/element-parameters.md

```diff
@@ -261,8 +301,152 @@ The components of this group are:
 ```
 The possible values of the optional `direction` switch are:
 ```{code} yaml
-FORWARD             # Injected particle propagates in forward direction. Default.
+FORWARDS            # Injected particle propagates in forward direction. Default.
 BACKWARDS           # Injected particle propagates in reverse direction.
 ```
 
 See the [](#s:forking) chapter for more details.
+
+%---------------------------------------------------------------------------------------------------
+(s:girder.params)=
+## GirderP: Girder Parameters
+
+In Construction...
+
+%---------------------------------------------------------------------------------------------------
+(s:init.particle.params)=
+## InitialParticleP: Initial Particle Coordinates Parameters
+
+In Construction...
+
+%---------------------------------------------------------------------------------------------------
+(s:mag.mult.params)=
+## MagneticMultipoleP: Magnetic Multipole Parameters
+
+The `MagneticMultipoleP` parameter group describes magnetic multipoles associated with the lattice
+element. For a multipole of order `N`, the magnetic field {math}`(B_x, B_y)`
+at a point {math}`\bf r` in terms of the points polar coordinates {math}`(r, \theta)` is
+```{math}
+:label: bbmult
+
+B_x({\bf r}) + i \, B_y({\bf r}) = 
+\frac{1}{N!}(B_n + i \, B_s) \, e^{-i(N+1)T} \, e^{iN\theta} \, r^N
+```
+where {math}`T` is the "tilt" of the `N`{sup}`th` multipole and {Math}`B_n` and {math}`B_s` are
+the normal and skew components of the field. `N` is defined such that 
+{math}`N = 0` describes a dipolar multipole, {math}`N = 1` describes a quadrupolar multipole, etc.
+
+The three parameters, {math}`T`, {Math}`B_n` and {math}`B_s` are not independent and only two
+are needed to specify a multipole. 
+However, it is sometimes convenient to use all three. 
+For example, {Math}`B_n` and {math}`B_s` with {math}`N=0` can be used to describe 
+an element with independent horizontal and vertical steerings while {math}`T` can be used
+to represent rotational errors.
+
+The components of `MagneticMultipoleP` for specifying a multipolar field of order `N` is:
+```{code} yaml
+tiltN     - Tilt
+BnN       - Normal component 
+BsN       - Skew component
```

Comment:
This is not yaml formatted :)

We actually need to yaml format to see a few implications on discriminators for serialization/deserialization from/to yaml/json/toml. Will mention separately.